src/ChipCarousel/messages.js

```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  scrollToPrevious: {
+    id: 'pgn.ChipCarousel.scrollToPrevious',
+    defaultMessage: 'Scroll to previous',
+    description: 'Accessibility text describing the action for navigating a scrollable carousel to the previous element',
+  },
+  scrollToNext: {
```

Comment:
Also in other translations, I noticed that we use a slightly different format for recording the key for translations. I suggest sticking to uniformity.

`  'subscription.details.program.type.microMasters': {
    id: 'subscription.details.program.type.microMasters',
    defaultMessage: 'MicroMasters',
    description: 'Label to render the MicroMasters Certificate program type.',
  },`